.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+    "editor.formatOnPaste": true,
+    "editor.formatOnSave": true
+}
```

Comment:
Please do not upload your local configs (you can add it to .gitignore)

Reply:
omg this is humbling